src/App.css

```diff
@@ -1,3 +1,16 @@
 .App {
   text-align: center;
 }
+
+a {
+  font-style: bold;
+  text-decoration: none;
+}
+
+.container {
+  max-width: 1180px;
+  padding: 0 10px;
+  margin: 0 auto;
+  padding: 0 50px;
```

Comment:
This is fine, make sure to read a little about rem, em, vw etc units.
When to use them for padding margin etc